genaisrc/README-llmstxt-optimizer.md

```diff
@@ -0,0 +1,112 @@
+# LLM-Optimized Content Generator
+
+This GenAIScript analyzes markdown files from an Astro Starlight website and generates LLM-optimized content stored in the `llmstxt` frontmatter field.
+
+## Overview
+
+The script:
+1. Processes markdown (`.md`) and MDX (`.mdx`) files
+2. Analyzes the content (excluding frontmatter)
+3. Generates a concise, LLM-optimized version
+4. Stores the optimized content in the `llmstxt` frontmatter field
+5. Updates files in-place
+
+## Usage
+
+### Basic Usage
+
+```bash
+# Process a single file
+node packages/cli/dist/src/index.js run llmstxt-optimizer path/to/file.md
```

Comment:
Always using the "genaiscript" cli instead of the node way. End users use the cli